openfecwebapp/templates/legal-search-results.html

```diff
@@ -0,0 +1,78 @@
+{% extends "layouts/main.html" %}
+{% import 'macros/search.html' as search %}
+
+{% block title %}
+  Search results for "{{query}}"
+{% endblock %}
+
+{% block body %}
+<header class="page-header slab slab--primary">
+  <span class="page-header__title">Search results</span>
+    <form id="large-search" autocomplete="off" class="search__container js-search">
+      <div class="combo combo--search">
+        <select class="search__select" name="search_type">
```

Comment:
This `select` should have the class `.select--alt-primary` and the `button` should be `.button--standard`.